src/main/kotlin/se/uulm/snowballr/backend/model/IdentifierType.kt

```diff
@@ -8,4 +8,5 @@ package se.uulm.snowballr.backend.model
 enum class IdentifierType(val displayName: String) {
     ID("ID"),
     EMAIL("email"),
+    LOCAL_ID("localId"),
```

Comment:
```suggestion
    LOCAL_ID("local ID"),
```
To be consist with ID("ID")? Or do you think it is better to be consistent with the field name in the project paper object?

Reply:
I think it doesn't really matter here, but I guess it looks a bit nicer to be consistent with the ID.